draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -654,476 +526,437 @@
-   reuse its URI, even after that Trigger Status Resource has been
-   removed.</t>
+   </section>
 
+   <section title="Modifying Triggers" anchor="modifying-triggers">
    <t>
-   dCDN SHOULD track and report on the progress of CI/T Trigger
-   Commands using a Trigger Status Resource (<xref target="trigger-status-resources"/>). If the
-   dCDN is not able to do that, it MUST indicate that it has accepted
-   the request but will not be providing further status updates. To do
-   this, it sets the status of the Trigger Status Resource to
-   "processed". In this case, CI/T processing should continue as for a
-   "complete" request, so the Trigger Status Resource MUST be added to
-   dCDN's collection of complete Trigger Status Resources. dCDN
-   SHOULD also provide an estimated completion time for the request by
-   using the "etime" property of the Trigger Status Resource. This will
-   allow uCDN to schedule pre-positioning after an earlier delete of
-   the same URLs is expected to have finished.</t>
-
+	Modification of existing triggers is useful for uCDN to correct an error in trigger specification or trigger extension(s) 
+	that may govern when the trigger is to be processed.
+   </t>
    <t>
-   If dCDN can track the execution of CI/T Commands and a
-   CI/T Command is queued by dCDN for later action, the "status"
-   property of the Trigger Status Resource MUST be "pending". Once
-   processing has started, the status MUST be "active". Finally, once
-   the CI/T Command is complete, the status MUST be set to "complete" or
-   "failed".</t>
-
+	uCDN can request modification of an existing trigger resource by sending an updated trigger representation to the trigger URI 
+	using HTTP POST command.  
+   </t>
    <t>
-   A CI/T Trigger Command may result in no activity in dCDN if, for
-   example, it is an "invalidate" or "purge" request for datdCDN
-   has not yet acquired, or a "preposition" request for data that it has
-   already acquired and that is still valid. In this case, the status
-   of the Trigger Status Resource MUST be "processed" or "complete", and
-   the Trigger Status Resource MUST be added to dCDN's collection of
-   complete Trigger Status Resources.</t>
-
+	dCDN MAY accept modifications of the trigger specifications, trigger extensions and trigger labels, when the trigger is in 
+	"pending" state, i.e. dCDN didn't start its processing yet.
+   </t>
    <t>
-   Once created, Trigger Status Resources can be cancelled or deleted by
-   uCDN, but not modified. dCDN MUST reject PUT and POST
-   requests from uCDN to Trigger Status Resources by responding with
-   an appropriate HTTP status code -- for example, 405 ("Method Not Allowed").</t>
+	dCDN MAY also accept a request to change the trigger state subject to the following constraints:
+      	<list style="symbols">
+	<t>the requested state is "cancelled", and the trigger was in either "pending" or "active" state when dCDN receives the request</t>
```

Comment:
I feel like this is a leftover artifact.  The previous version didn't have a "Cancelling Triggers" section, and this section used to cover both cancellation and modification.

Now that there is a dedicated cancellation section, it seems like this bullet point belongs there, and it is a distraction to the discussion of modifying triggers.

Reply:
this is not, and here's why - in the current form, "cancellation" is a form of trigger modification (as opposed to trigger deletion, which like cancellation, stops the trigger processing, but also removes the resource). 
therefore i mention cancellation in the modification section but refer to the cancellation section for details on that.